### DISCUSSION TITLE
[Feature] Share Memories

### DISCUSSION THREAD

**Discussion by srwareham** [immich-app/immich#3473] [Feature Request]
### The feature

The memories feature is a wonderful addition. I often see photos with friends and would like to send them the photo to reminisce. It would be great if the memory view had the same share button as the typical photo view.

Thanks for the great project!

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by enese94** [comment]
It would be really great, Or put a shortcut to the original file.

**Comment by heikomat** [comment]
My girlfriend would love to be able to share photos from the memory lane without taking a screenshot of it.

It seems like a couple of similar feature-requests exist:
- https://github.com/immich-app/immich/discussions/4496
- https://github.com/immich-app/immich/discussions/4950

But this one is the most upvoted yet, probably because it focuses on a single aspect (sharing the current image)

**Comment by bo0tzz** [comment]
Closing in favour of #4496
